fw/control.c

```diff
@@ -127,6 +140,10 @@ int8_t control_run_message(control_message* m) //{{{
 			uart_tx_dma();
 			ret = -1;
 		break;
+		case CONTROL_TYPE_GET_MODE_PORTABLE:
```

Comment:
Remember to implement the software side to not let the user download a specific file if the BattOr is in portable mode.